config/routes.rb

```diff
@@ -1,3 +1,6 @@
 Rails.application.routes.draw do
-  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
-end
+  resources :centers
+  get '/index' => 'centers#index'
+  get '/centers/:id' => 'centers#show'
```

Comment:
can remove the `get` statement, these are already given by the `resources :centers`